pkg/services/alerting/notifiers/hipchat_test.go

```diff
@@ -27,12 +33,7 @@ func TestHipChatNotifier(t *testing.T) {
 			})
 
 			Convey("from settings", func() {
-				json := `
-				{
-          "url": "http://google.com"
-				}`
-
-				settingsJSON, _ := simplejson.NewJson([]byte(json))
+				settingsJSON, _ := simplejson.NewJson([]byte(jsonGoogle))
```

Comment:
I'd suggest reverting this change, having the test data close to where it is being used makes the code harder to read. If we were to use the same test data in multiple locations it may have made sense to move it to the top level.